src/librustc/traits/error_reporting.rs

```diff
@@ -949,6 +949,11 @@ impl<'a, 'gcx, 'tcx> InferCtxt<'a, 'gcx, 'tcx> {
                         .map_or(false, |sized_id| sized_id == trait_ref.def_id())
                     {
                         self.need_type_info(body_id, span, self_ty);
+                    } else if
```

Comment:
nit: preexisting, but I think
```Rust
if self.tcx.lang_items().move_trait() == Some(trait_def_id) ||
    self.tcx.lang_items().sized_trait() == Some(trait_def_id)
```

is the preferred style.

Reply:
Fixed.